tools/test/h5dump/testh5dump.sh.in

```diff
@@ -537,6 +537,69 @@ TOOLTEST() {
 
 }
 
+# same as TOOLTEST but masks out the compression values
+TOOLTESTC() {
+    # check if caseless compare and diff requested
+    if [ "$1" = ignorecase ]; then
```

Comment:
Seems like this whole block isn't needed since the `$1` argument is supposed to be the leading number in the compression ratio to not mask out

Reply:
This was a weird way to add an optional parameter, which controls whether diff adds a "-i".
I left it as is.

Reply:
We should probably remove this block because the script doesn't account for this being `$1` and the mask part to ignore being `$2`, it's just checking `$1` for both of them  which may be confusing to some and `ignorecase` will never be passed for `$1`

Reply:
No look athe the if/else  if it matches it shifts otherwise continues

Reply:
Ah ok, missed the shift. I guess that's fine then, even though nothing's using it currently.